File: phone-append-2-nodejs-examples/business_phone_append_rest_example.js
import { BusinessPhoneAppendClient } from '../phone-append-2-nodejs/REST/business_phone_append_rest.js';

async function businessPhoneAppendRestExample(isLive, licenseKey) {
    console.log("\r\n-------------------------------------------------");
    console.log("PhoneAppend2 - BusinessPhoneAppend - REST Example");
    console.log("-------------------------------------------------");

    const businessName = "Brooks Institute";
    const address = "27 E Cota ST";
    const city = "Santa Barbara";
    const state = "CA";
    const postalCode = "93101";

    console.log("\r\n* Input *\r\n");
    console.log(`Business Name: ${businessName}`);
    console.log(`Address      : ${address}`);
    console.log(`City         : ${city}`);
    console.log(`State        : ${state}`);
    console.log(`Postal Code  : ${postalCode}`);
    console.log(`License Key  : ${licenseKey}`);
    console.log(`Is Live      : ${isLive}`);

    try {
        const response = await BusinessPhoneAppendClient.invokeAsync(
            businessName,
            address,
            city,
            state,
            postalCode,
            licenseKey,
            isLive
        );

        console.log("\r\n* Phone Info *\r\n");
        if (response && !response.Error) {
            if (response.PhoneInfo) {
                console.log(`Phone         : ${response.PhoneInfo.Phone}`);
                console.log(`Name          : ${response.PhoneInfo.Name}`);
                console.log(`Address       : ${response.PhoneInfo.Address}`);
                console.log(`City          : ${response.PhoneInfo.City}`);
                console.log(`State         : ${response.PhoneInfo.State}`);
                console.log(`Postal Code   : ${response.PhoneInfo.PostalCode}`);
                console.log(`Is Residential: ${response.PhoneInfo.IsResidential}`);
                console.log(`Certainty     : ${response.PhoneInfo.Certainty}`);
                console.log(`Line Type     : ${response.PhoneInfo.LineType}`);
            } else {
                console.log("No phone info found.");
            }
        } else {
            console.log("No phone info found.");
        }

        if (response.Error) {
            console.log("\r\n* Error *\r\n");
            console.log(`Error Type     : ${response.Error.Type}`);
            console.log(`Error Type Code: ${response.Error.TypeCode}`);
            console.log(`Error Desc     : ${response.Error.Desc}`);
            console.log(`Error Desc Code: ${response.Error.DescCode}`);
        }
    } catch (e) {
        console.log("\r\n* Error *\r\n");
        console.log(`Error Message: ${e.message}`);
    }
}

export { businessPhoneAppendRestExample };

File: phone-append-2-python-examples/business_phone_append_rest_example.py
import sys
import os

sys.path.insert(0, os.path.abspath("../phone-append-2-python/REST"))

from business_phone_append_rest import get_business_phone_append

def business_phone_append_rest_sdk_go(is_live: bool, license_key: str) -> None:
    print("\r\n-------------------------------------------------")
    print("PhoneAppend2 - BusinessPhoneAppend - REST Example")
    print("-------------------------------------------------")

    business_name = "Brooks Institute"
    address = "27 E Cota ST"
    city = "Santa Barbara"
    state = "CA"
    postal_code = "93101"

    print("\r\n* Input *\r\n")
    print(f"Business Name: {business_name}")
    print(f"Address      : {address}")
    print(f"City         : {city}")
    print(f"State        : {state}")
    print(f"Postal Code  : {postal_code}")
    print(f"License Key  : {license_key}")
    print(f"Is Live      : {is_live}")

    try:
        response = get_business_phone_append(business_name, address, city, state, postal_code, license_key, is_live)

        print("\r\n* Phone Info *\r\n")
        if response and not response.Error:
            if response.PhoneInfo:
                print(f"Phone         : {response.PhoneInfo.Phone}")
                print(f"Name          : {response.PhoneInfo.Name}")
                print(f"Address       : {response.PhoneInfo.Address}")
                print(f"City          : {response.PhoneInfo.City}")
                print(f"State         : {response.PhoneInfo.State}")
                print(f"Postal Code   : {response.PhoneInfo.PostalCode}")
                print(f"Is Residential: {response.PhoneInfo.IsResidential}")
                print(f"Certainty     : {response.PhoneInfo.Certainty}")
                print(f"Line Type     : {response.PhoneInfo.LineType}")
            else:
                print("No phone info found.")
        else:
            print("No phone info found.")

        if response.Error:
            print("\r\n* Error *\r\n")
            print(f"Error Type     : {response.Error.Type}")
            print(f"Error Type Code: {response.Error.TypeCode}")
            print(f"Error Desc     : {response.Error.Desc}")
            print(f"Error Desc Code: {response.Error.DescCode}")

    except Exception as e:
        print("\r\n* Error *\r\n")
        print(f"Error Message: {str(e)}")